tests/unit/test_pyramid_app/app_test.py

```diff
@@ -15,5 +17,26 @@ def test_index():
     assert app.index(sentinel.request) == {"Hello": "Pyramid!"}
 
 
-def test_status():
-    assert app.status(sentinel.request) == {"status": "okay"}
+def test_status(capture_message):
+    request = DummyRequest(path="/_status")
+
+    assert app.status(request) == {"status": "okay"}
+
+    capture_message.assert_not_called()
+
+
+def test_status_sends_test_messages_to_sentry(capture_message):
+    request = DummyRequest(params={"sentry": ""}, path="/_status")
+
+    app.status(request)
+
+    capture_message.assert_called_once_with(
+        "Test message from test-pyramid-app's status view"
+    )
+
+
+@pytest.fixture(autouse=True)
+def capture_message(mocker):
+    return mocker.patch(
+        "test_pyramid_app.app.capture_message", autospec=True, spec_set=True
+    )
```

Comment:
The one downside of pytest-mock compared to our own `patch` fixture is that you have to pass `autospec=True` and `spec_set=True`, whereas our `patch` fixture does that automatically.

It might be possible for us to add this to pytest-mock, maybe you can just do this in `tests/conftest.py`:

```python
@pytest.fixture
def mocker(mocker):
    def call_mocker(*args, **kwargs):
        kwargs.setdefault("autospec", True)
        kwargs.setdefault("spec_set", True)
        return mocker(*args, **kwargs)

    return call_mocker